cross/fuse/digests

```diff
@@ -0,0 +1,6 @@
+fuse-2.9.4.tar.gz SHA1 c8b25419f33624dc5240af6a5d26f2c04367ca71
+fuse-2.9.4.tar.gz SHA256 6be9c0bff6af8c677414935f31699ea5a7f8f5f791cfa5205be02ea186b97ce1
+fuse-2.9.4.tar.gz MD5 ecb712b5ffc6dffd54f4a405c9b372d8
+fuse-2.9.4.tar.gz SHA1 c8b25419f33624dc5240af6a5d26f2c04367ca71
```

Comment:
Please remove the current digests file and recreate, or manually remove the double entries. I think you ran `make digests` twice.